torchft/process_group.py

```diff
@@ -615,8 +645,14 @@ def _worker(
         try:
             store = create_store_client(store_addr)
 
-            # pyre-fixme[20]: expects argument options
-            pg = cls.PG_CLASS(store, rank, world_size)
+            try:
+                # pyre-fixme[20]: expects argument options
+                pg = cls.PG_CLASS(store, rank, world_size)
+            except Exception as e:
+                logger.exception(f"got exception in worker: {e}")
+                tx.put(e)
+                return
+            tx.put(None)
```

Comment:
Since PGWrapper now uses _create_pg instead of PG_CLASS instantiation, should we make the code more consistent by having `_create_pg` here too?